utils/router/src/utils/cannon.ts

```diff
@@ -37,16 +39,22 @@ export type Config = JTDDataType<typeof config>;
 // if not deployed, deploy the specified hardhat contract with specfied options, export
 // address, abi, etc.
 // if already deployed, reexport deployment options for usage downstream and exit with no changes
-const routerAction = {
+export default {
+  label: 'router',
+
   validate: config,
 
-  async getState(runtime: ChainBuilderRuntimeInfo, ctx: ChainBuilderContext, config: Config) {
-    if (!runtime.baseDir) {
+  async getState(
+    runtime: ChainBuilderRuntimeInfo,
+    ctx: ChainBuilderContextWithHelpers,
+    config: Config
+  ) {
+    if (!(runtime as unknown as { baseDir: string }).baseDir) {
       return null; // skip consistency check
       // todo: might want to do consistency check for config but not files, will see
     }
 
-    const newConfig = this.configInject(ctx, config);
+    const newConfig = this.configInject(ctx, config, null as unknown as PackageState);
```

Comment:
btw, `packageSTate` is actually passed into the `getState` function now, shuoldn'th ave to use `null`